diff -urN configparser-3.5.0.org/setup.py configparser-3.5.0/setup.py
--- configparser-3.5.0.org/setup.py	2016-05-21 10:14:22.000000000 +0200
+++ configparser-3.5.0/setup.py	2018-11-30 11:17:12.134505201 +0100
@@ -42,7 +42,6 @@
     py_modules=modules,
     package_dir={'': 'src'},
     packages=find_packages('src'),
-    namespace_packages=['backports'],
     include_package_data=True,
     zip_safe=False,
     install_requires=requirements,
